LEAF_Request_Portal/templates/reports/LEAF_mass_action.tpl

```diff
@@ -89,11 +95,818 @@
+    }
+
+    if (!getSubmitted) {
+        requestQuery.terms.push({
+            id: "stepID",
+            operator: "!=",
+            match: "submitted",
+        });
+    }
+
+    if (getReminder) {
+        requestQuery.joins.push("action_history");
+        requestQuery.terms.push({
+            id: "stepID",
+            operator: "!=",
+            match: "resolved",
+        });
+    }
+
+    if (getAction !== "") {
+        let stepID = $("select#step_select").val();
+        let dependencyID = $("select#requirements_select").val();
+        let formID = $("select#form_select").val();
+        let categoryID = formID.split("-")[0];
+
+        requestQuery.terms.push({
+            id: "categoryID",
+            operator: "=",
+            match: categoryID,
+            gate: "AND",
+        });
+
+        requestQuery.terms.push({
+            id: "stepID",
+            operator: "=",
+            match: stepID,
+            gate: "AND",
+        });
+
+        requestQuery.terms.push({
+            id: "dependencyID",
+            operator: "=",
+            indicatorID: dependencyID,
+            gate: "AND",
+        });
+    }
+
+    //handle extraTerms
+    let isJSON = true;
+    let advSearch = {};
+    try {
+        advSearch = $.parseJSON(extraTerms);
+    } catch (err) {
+        isJSON = false;
+    }
+
+    if (isJSON) {
+        requestQuery.terms = $.merge(requestQuery.terms, advSearch);
+    } else if (typeof extraTerms === "string") {
+        requestQuery.terms.push({
+            id: "title",
+            operator: "LIKE",
+            match: "*" + extraTerms.trim() + "*",
+        });
+    }
+
+    return requestQuery;
+}
+
+/**
+ * Looks up requests based on filter/searchbar and builds table with the results
+ *
+ * @param {Object}                [queryObj]                                                Object to pass to form/query
+* @param {Integer} [thisSearchID]                When done() is called, this param is compared to the global searchID. If they are not equal, then the results are not processed.
+* @param {Number}                [getReminder]                 Number of days for email reminder selection
+*/
+function listRequests(queryObj, thisSearchID, getReminder = 0) {
+    $("#searchResults").hide();
+    $("#errorMessage").hide();
+    $("table#requests tr.requestRow").remove();
+    $("#iconBusy").show();
+
+    $.ajax({
+        type: "GET",
+        url: "./api/form/query",
+        data: { q: JSON.stringify(queryObj), CSRFToken: massActionToken },
+        cache: false,
+    })
+        .done(function (data) {
+            if (thisSearchID === searchID) {
+                if (Object.keys(data).length) {
+                    let totalCount = 0;
+
+                    $.each(data, function (index, value) {
+                        let displayRecord = true;
+                        // If this is email reminder list, then compare against give time period
+                        if (getReminder) {
```

Comment:
getReminder is a number here.

Reply:
This was just copied in. I did update it to be > 0